packages/core/src/components/Card.story.tsx

```diff
@@ -5,14 +5,17 @@ import IconChevronRight from '@airbnb/lunar-icons/lib/interface/IconChevronRight
 import LoremIpsum from ':storybook/components/LoremIpsum';
 import lunar from ':storybook/images/lunar-logo.png';
 import stars from ':storybook/images/stars.jpg';
-import moon from ':storybook/images/moon.png';
 import Text from './Text';
 import ResponsiveImage from './ResponsiveImage';
 import Card, { Content } from './Card';
 
+const moon =
```

Comment:
it was worth a shot to try this :( thank you!

let's remove it/ go back to `import moon from ':storybook/images/moon.png';`

and call this good to go!